GenderPayGap.WebUI/Controllers/Admin/AdminDownloadsController.cs

```diff
@@ -60,6 +60,35 @@ public FileContentResult DownloadUserConsentSendUpdates()
             return fileContentResult;
         }
 
+        [HttpGet("downloads-new/user-consent-allow-contact-for-feedback")]
+        public FileContentResult DownloadUserConsentAllowContactForFeedback()
+        {
+            List<User> users = dataRepository.GetAll<User>()
+                .Where(user => user.Status == UserStatuses.Active)
+                .Where(user => user.UserSettings.Any(us => us.Key == UserSettingKeys.AllowContact && us.Value.ToLower() == "true"))
+                .ToList();
+
+            var records = users.Select(
+                    u => new {
+                        u.Firstname,
+                        u.Lastname,
+                        u.JobTitle,
+                        u.EmailAddress,
+                        u.ContactFirstName,
+                        u.ContactLastName,
+                        u.ContactJobTitle,
+                        u.ContactEmailAddress,
+                        u.ContactPhoneNumber,
+                        u.ContactOrganisation
+                    })
+                .ToList();
+
+            string fileDownloadName = $"Gpg-UserConsent-AllowContactForFeedback-{DateTime.Now:yyyy-MM-dd HH:mm}.csv";
```

Comment:
Should these be `VirtualDateTime.Now`?

Reply:
It doesn't really matter - but it's probably best to use `VirtualDateTime` so it's easy to see that we haven't missed any